tests/test_autonames/conftest.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+
+@pytest.fixture()
+def ignore_migration(settings):
+    """We patch settings to ignore this one bad migration."""
+    settings.DTM_IGNORED_MIGRATIONS = {
+        ('main_app', '0004_auto_20191119_2125'),
+    }
+    return settings
```

Comment:
I think having this directly in the test (instead of a fixture in another file) would be clearer.

Reply:
Fair enough! Scheduled: https://github.com/wemake-services/django-test-migrations/issues/36